src/PurgeClient/VarnishPurgeClient.php

```diff
@@ -80,6 +80,23 @@ public function purgeAll()
         );
     }
 
+    /**
+     * Adds a Host header for Purge.
+     *
+     * @param array $headers
+     * @return array
+     */
+    private function addHostHeader(array $headers)
+    {
+        $server = $this->configResolver->getParameter('http_cache.purge_servers')[0];
+        $host = parse_url($server, PHP_URL_HOST);
+        if ($host) {
+            $headers['Host'] = $host;
+        }
+
+        return $headers;
```

Comment:
`$_SERVER['SERVER_NAME']` was added as fallback in https://github.com/ezsystems/ezplatform-http-cache/commit/6bcd8833bd486b345b363328a114b84654e9da9a for some reason ( I don't remember the case exactly, but it was needed). Here that fallback seems to have been removed

Reply:
Actually, when thinking about it.. What would make sense is to use `http_cache.purge_servers` by default; if set ( and possibly not '' ? ), then use $_SERVER['SERVER_NAME'] as fallback.. In our current implementation it is the other way around...

Reply:
@vidarl can you please check https://github.com/ezsystems/ezplatform-http-cache/pull/87/commits/37675f0806a2a409252c4a886485975b948ff45b?

Reply:
Looks about right to me 👍